src/playback/internal/playbackcontroller.cpp

```diff
@@ -728,6 +732,13 @@ void PlaybackController::toggleMidiInput()
     notifyActionCheckedChanged(MIDI_ON_CODE);
 }
 
+void PlaybackController::setMidiInputPitch(bool useWrittenPitch)
```

Comment:
```suggestion
void PlaybackController::setMidiUseWrittenPitch(bool useWrittenPitch)
```
would be slightly clearer (sorry for yet another comment 🙂)